conf/messages.en

```diff
@@ -464,6 +464,9 @@ soleTraderEntityVerification.failure.detail = Check your Self Assessment records
 
 duplicateSubscription.title = Your organisation is already registered
 duplicateSubscription.detail = Someone in {0} has already registered for Plastic Packaging Tax (PPT).
+duplicateSubscription.account.detail.link = Go to your PPT account to:
```

Comment:
Hi Carl, I am not sure if you have seen my previous comments, but I asked Lisa about the brackets and she said to leave them in. But fell free to ask her agin as she might misunderstood me.

Reply:
Ah I hadn't. Changed back 👍